src/Explorer/Graph/GraphExplorerComponent/NodePropertiesComponent.tsx

```diff
@@ -45,6 +45,7 @@ export interface NodePropertiesComponentProps {
   selectNode: (id: string) => void;
   updatePossibleVertices: () => Q.Promise<PossibleVertex[]>;
   possibleEdgeLabels: Item[];
+  //eslint-disable-next-line
```

Comment:
Is there no other way to make eslint happy without disabling it?

Reply:
revert change